README.md

```diff
@@ -811,6 +813,23 @@ RuboCop::RakeTask.new(:rubocop) do |task|
 end
 ```
 
+## Caching
+
+Large projects containing hundreds or even thousands of files can take
+a really long time to inspect, but RuboCop has functionality to
+mitigate this problem. A result caching mechanism is activated if the
```

Comment:
I think the term "result" is kind of confusing/unclear. I'd suggest elaborating on this.